examples/hello-cpp-server-redis/main.cpp

```diff
@@ -0,0 +1,102 @@
+#include <launchdarkly/context_builder.hpp>
+#include <launchdarkly/server_side/client.hpp>
+#include <launchdarkly/server_side/config/config_builder.hpp>
+
+#include <launchdarkly/server_side/integrations/redis/redis_source.hpp>
+
+#include <cstring>
+#include <iostream>
+
+// Set SDK_KEY to your LaunchDarkly SDK key.
+#define SDK_KEY ""
+
+// Set FEATURE_FLAG_KEY to the feature flag key you want to evaluate.
+#define FEATURE_FLAG_KEY "my-boolean-flag"
+
+// Set INIT_TIMEOUT_MILLISECONDS to the amount of time you will wait for
+// the client to become initialized.
+#define INIT_TIMEOUT_MILLISECONDS 3000
+
+char const* get_with_env_fallback(char const* source_val,
+                                  char const* env_variable,
+                                  char const* error_msg);
+using namespace launchdarkly;
+using namespace launchdarkly::server_side;
+
+int main() {
+    char const* sdk_key = get_with_env_fallback(
+        SDK_KEY, "LD_SDK_KEY",
+        "Please edit main.c to set SDK_KEY to your LaunchDarkly server-side "
+        "SDK key "
+        "first.\n\nAlternatively, set the LD_SDK_KEY environment "
+        "variable.\n"
+        "The value of SDK_KEY in main.c takes priority over LD_SDK_KEY.");
+
+    auto config_builder = ConfigBuilder(sdk_key);
+
+    using LazyLoad = server_side::config::builders::LazyLoadBuilder;
+
+    auto redis = integrations::RedisDataSource::Create("redis://localhost:6379",
```

Comment:
Maybe a comment here instructing them to set this URL to something that will work for them.